pages/cockpit/quick-start/get-started.adoc

```diff
@@ -24,6 +24,8 @@ This dashboard gives you an overview of your account hierarchy.
 
 You can create one or more organizations within your account and one or more environments within your organizations. Each environment can then be linked to existing APIM or AM installations.
 
+image::cockpit/cockpit-add-organizations-or-environments.png[]
```

Comment:
I think this image should replace the image L21, because this is now the new home page. And we can keep showing the one L21 but with a sentence explaining that this view is accessible when clicking on the 2nd link.